Use gitty for loading code and JSON.

rec opened the pull request:
Fixes #200.

adammhaile commented:
oops... jumped the gun. I approved before checks ran. 
Looks like you aren't loading gitty for the tests

rec commented:
How weird that requirements.txt isn't loaded during tests!  Added it to test_requirements.txt, pushed it as a fixup commit, CI passed, now rebased and pushed - likely all will be green when you see this.

adammhaile commented:
Cool

On Jun 11, 2017 4:15 PM, "Tom Swirly" <notifications@github.com> wrote:

> How weird that requirements.txt isn't loaded during tests! Added it to
> test_requirements.txt, pushed it as a fixup commit, CI passed, now rebased
> and pushed - likely all will be green when you see this.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/ManiacalLabs/BiblioPixel/pull/211#issuecomment-307653854>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AA6a6u2feob_0dAYEzkmCBHa2j_Eo3rBks5sDErUgaJpZM4N2fNR>
> .
>
